dev/InfoBar/TestUI/InfoBar.cs

```diff
@@ -199,33 +205,21 @@ public object ActionButtonCommandParameter
         /* Close Button Properties
          * 
          */
-        public object CloseButtonContent
-        {
-            get { return (object)GetValue(CloseButtonContentProperty); }
-            set { SetValue(CloseButtonContentProperty, value); }
-        }
-
-        public static readonly DependencyProperty CloseButtonContentProperty =
-            DependencyProperty.Register(nameof(CloseButtonContent), typeof(object), typeof(InfoBar), new PropertyMetadata(null, OnPropertyChanged));
-
-        public Style CloseButtonStyle
```

Comment:
>CloseButtonStyle [](start = 21, length = 16)

@gabilka@microsoft.com sorry if we already went over this, but why is the style property being removed?

Reply:
I think initially @gabbybilka  thought CloseButtonStyle was solely related to the user-custom close button (which has since been removed). I think Gabby does want to add the property back in, so that users can restyle the x close button. 

I added CloseButtonStyle back in with my most recent push, and if needed, we can just delete it again.

Reply:
Thank you @chenss3, apologies for the late response. That is my intention!